src/app/pages/contact/contact.component.html

```diff
@@ -1,6 +1,21 @@
-<div class="container pt-3">
-  <h1 id="title">Contact Us</h1>
+<div class="container">
+  <h1 class="mt-3" id="title">Contact Us</h1>
   <br>
+<<<<<<< HEAD
```

Comment:
Uhhhhhhhhhhhh why is there a merge conflict in here lol.

Reply:
Seems like this contact us page is out of date with the one where it mentions our Facebook and Instagram (I think that one was on static-components branch? I'm not sure). Wouldn't want conflicting merges into master, so check if everything works together..

Reply:
```suggestion
```